analysis/cnvs/query_match_nch_clinvar_cnvs.ipynb

```diff
@@ -632,12 +632,46 @@
    "execution_count": 13,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "def canonicalize(intervals: list) -> list:\n",
+    "    \"\"\"in: list of start/stop tuples\n",
+    "    out: list of deduplicated, sorted, consolidated start/stop tuples\n",
```

Comment:
@jarbesfeld This should be updated to follow the convention we use (sphinx-like)